wsit/hudson/jenkins.sh

```diff
@@ -0,0 +1,361 @@
+#!/bin/bash -ex
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+# Copyright (c) 2012-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# The contents of this file are subject to the terms of either the GNU
+# General Public License Version 2 only ("GPL") or the Common Development
+# and Distribution License("CDDL") (collectively, the "License").  You
+# may not use this file except in compliance with the License.  You can
+# obtain a copy of the License at
+# https://oss.oracle.com/licenses/CDDL+GPL-1.1
+# or LICENSE.txt.  See the License for the specific
+# language governing permissions and limitations under the License.
+#
+# When distributing the software, include this License Header Notice in each
+# file and include the License file at LICENSE.txt.
+#
+# GPL Classpath Exception:
+# Oracle designates this particular file as subject to the "Classpath"
+# exception as provided by Oracle in the GPL Version 2 section of the License
+# file that accompanied this code.
+#
+# Modifications:
+# If applicable, add the following below the License Header, with the fields
+# enclosed by brackets [] replaced by your own identifying information:
+# "Portions Copyright [year] [name of copyright owner]"
+#
+# Contributor(s):
+# If you wish your version of this file to be governed by only the CDDL or
+# only the GPL Version 2, indicate your decision by adding "[Contributor]
+# elects to include this software in this distribution under the [CDDL or GPL
+# Version 2] license."  If you don't indicate a single choice of license, a
+# recipient has the option to distribute your version of this file under
+# either the CDDL, the GPL Version 2 or to extend the choice of license to
+# its licensees as provided above.  However, if you add GPL Version 2 code
+# and therefore, elected the GPL Version 2 license, then the option applies
+# only if the new code is made subject to such option by the copyright
+# holder.
+#
+
+#set env like CTS_URL [required] , MVN_SETTINGS[optional] , GF_URL [optional]
+METRO_MAJOR_VERSION=2.4.0
+WORKSPACE=$PWD  #linux
+METRO_REVISION=$BRANCH_NAME
+export WORK_DIR=$WORKSPACE
+echo "WORK_DIR: $WORK_DIR"
+export GF_SVN_ROOT=$WORK_DIR/glassfish
+export DTEST_SVN_ROOT=$GF_SVN_ROOT/appserver/tests/appserv-tests
+export METRO_SVN_ROOT=$WORK_DIR/metro-wsit
+
+
+cd $METRO_SVN_ROOT/wsit/hudson/
+source common.sh
+if [ -z "$M2_LOCAL_REPO" ]; then
+    export M2_LOCAL_REPO=$WORK_DIR/.repository
+fi
+
+#PROXY
+set_proxy
+
+export JAVA_MEMORY_PROP="-Xms256m -Xmx768m -XX:PermSize=256m -XX:MaxPermSize=512m"
```

Comment:
```-XX:PermSize=256m -XX:MaxPermSize=512m``` were deprecated on jdk8 and may cause troubles going forward... better to remove them (...you should see some warning on JDK8)